src/ga.js

```diff
@@ -194,10 +194,10 @@ function sendBidRequestToGa(bid) {
 
 function sendBidResponseToGa(bid) {
 
-	if (bid && bid.bidder) {
+	if (bid && bid.bidderCode) {
```

Comment:
bidResponse event is triggered before bid.bidder is set on bidmanager.addBidResponse function.